<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogExport</class>
 <widget class="QDialog" name="DialogExport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export simulation results</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QComboBox" name="cmbTimepoint"/>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="radSingleTimepoint">
       <property name="text">
        <string>Concentration image: &amp;single timepoint</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="radAllTimepoints">
       <property name="text">
        <string>Concentration images : &amp;all timepoints</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QRadioButton" name="radCSV">
       <property name="text">
        <string>CSV text file</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>radSingleTimepoint</tabstop>
  <tabstop>cmbTimepoint</tabstop>
  <tabstop>radAllTimepoints</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
